packages/bee-q/src/components/toast/scss/bq-toast.scss

```
@@ -0,0 +1,44 @@
+/* -------------------------------------------------------------------------- */
+/*                                Toast styles                */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-toast.variables';
+
+:host {
+  @apply inline-block;
```

The final touch will be to add some animation. To do so, we can:

```ts
// Extend keyframes and animation in tailwind.config.ts
  ...
  theme: {
    extend: {
      ...
      spacing: {  ... },
      keyframes: {
        'fade-in': {
          '0%': { opacity: '0' },
          '100%': { opacity: '1' },
        },
        'slide-up': {
          '0%': {
            transform: 'translateY(10px)',
          },
        },
      },
      animation: {
        'slide-in': 'fade-in 0.3s ease, slide-up 0.3s ease',
      },
    },
  },
```

and then here in the component styles:

```suggestion
  @apply inline-block animate-slide-in will-change-transform;
```

---

Done